src/main/java/dao/impl/OperatorForDbMap.java

```diff
@@ -0,0 +1,7 @@
+package dao.impl;
+
+import dao.OperationsWithMapDb;
+
+public class OperatorForDbMap {
+    public static final OperationsWithMapDb operator = new OperationsWithMapDbImpl();
```

Comment:
Don't use static for field for holding service classes, it is better to create one instance in the main method and pass it to the constructor everywhere you need